src/Csv.php

```diff
@@ -1,12 +1,13 @@
 <?php
+
 namespace ParseCsv;
 
 use ParseCsv\extensions\DatatypeTrait;
 
 class Csv {
 
     /*
-    Class: parseCSV v0.4.3 beta
+    Class: ParseCSV v1.0.0
```

Comment:
Again regarding Semantic Versioning, this line would read:

    Class: ParseCSV 1.0.0-rc.1

Note the removal of the `v` prefix.